trusspy/tools/helper_functions2.py

```diff
@@ -21,11 +21,11 @@ def __init__(self, xs, ys, zs, *args, **kwargs):
         FancyArrowPatch.__init__(self, (0,0), (0,0), *args, **kwargs)
         self._verts3d = xs, ys, zs
 
-    def draw(self, renderer):
+    def do_3d_projection(self, renderer=None):
```

Comment:
Originally, I took the code more or less from [StackOverflow](https://stackoverflow.com/questions/11140163/plotting-a-3d-cube-a-sphere-and-a-vector-in-matplotlib) (at least, I think so...). Your solution is more or less described [in a Notebook on Github](https://gist.github.com/WetHat/1d6cd0f7309535311a539b42cccca89c).

Reply:
Great, this works!